types/google-translate-api/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"semicolon": false
```

Comment:
Quick nit: Can you re-enable the rule? I know it's mostly personal preference, but it saves us some trouble long-term if all packages run with the same linting options.

Reply:
I have added semicolons in eb1881e8ab6f9aaf2934ab0a171ace0d9d5fde11.